### PR TITLE
Modify variable names related to scan angle bias correction

### PR DESCRIPTION
**Pull request by emilyhcliu**
Modify variable names related to scan-angle bis correction due to changes in UFO (https://github.com/JCSDA/ufo/pull/966)

This obs file changes for amsua_n19 is related to UFO PR 

Examples:
scan_angle_4th_order_1@GsiObsBiasTerm ---> scan_angle_order_4_1@GsiObsBiasTerm
scan_angle_4th_order_1@GSiObsBiasPredictor ---> scan_angle_order_4_1@GsiObsBiasPredictor

scan_angle_3rd_order_1@GsiObsBiasTerm ---> scan_angle_order_3_1@GsiObsBiasTerm
scan_angle_3rd_order_1@GSiObsBiasPredictor ---> scan_angle_order_3_1@GsiObsBiasPredictor

scan_angle_2nd_order_1@GsiObsBiasTerm ---> scan_angle_order_2_1@GsiObsBiasTerm
scan_angle_2nd_order_1@GSiObsBiasPredictor ---> scan_angle_order_2_1@GsiObsBiasPredictor

scan_angle_1st_order_1@GsiObsBiasTerm ---> scan_angle_order_1_1@GsiObsBiasTerm
scan_angle_1st_order_1@GSiObsBiasPredictor ---> scan_angle_order_1_1@GsiObsBiasPredictor

### CODE REVIEW

File: src/gsi-ncdiag/gsi_ncdiag.py [hunk 1]
```diff
@@ -1111,22 +1086,23 @@ def toIODAobs(self, OutDir, ObsBias, QCVars, TestRefs, clobber=True):
...
+                    "scan_angle_order_4_{:d}".format(a),
+                    "scan_angle_order_3_{:d}".format(a),
+                    "scan_angle_order_2_{:d}".format(a),
+                    "scan_angle_{:d}".format(a),
                 ]
                 ibc = 0
                 for vbc in valuebc:
                     varDict[vbc]['bctKey'] = vbc, writer.ObiastermName()
                     varDict[vbc]['bcpKey'] = vbc, writer.ObiaspredName()
+                    print("emily checking valuebc 1 = ", valuebc)
```
Comment (CoryMartin-NOAA) [inline src/gsi-ncdiag/gsi_ncdiag.py:1105]: Remove this comment

File: src/gsi-ncdiag/gsi_ncdiag.py [hunk 2]
```diff
@@ -1279,6 +1255,7 @@ def toIODAobs(self, OutDir, ObsBias, QCVars, TestRefs, clobber=True):
                     # store values in output data dictionary
                     outdata[varDict[value]['bctKey']] = obsbiastermsub
                     outdata[varDict[value]['bcpKey']] = obsbiaspredsub
+                    print("emily checking valuebc 2 = ", valuebc)
```
Comment (CoryMartin-NOAA) [inline src/gsi-ncdiag/gsi_ncdiag.py:1258]: Another debug comment to remove

File: src/gsi-ncdiag/gsi_ncdiag.py (hunk 1 above)
Comment (emilyhcliu) [reply src/gsi-ncdiag/gsi_ncdiag.py:1105]: @CoryMartin-NOAA  Oop!  Thanks for catching this. Fix is on the way...

**Comment by emilyhcliu** [comment]
@CoryMartin-NOAA @srherbener Thanks for reviewing this PR

**Comment by srherbener** [comment]
@emilyhcliu, no problem. Thanks for contributing!

**Pull request merged**